packages/kit/src/runtime/client/client.js

```diff
@@ -69,9 +69,10 @@ function native_navigation(url) {
 /**
  * @param {import('./types.js').SvelteKitApp} app
  * @param {HTMLElement} target
+ * @param {boolean} fine_grained_search_params_invalidation
  * @returns {import('./types.js').Client}
  */
-export function create_client(app, target) {
+export function create_client(app, target, fine_grained_search_params_invalidation) {
```

Comment:
I don't see a reason to pass this create_client as a param. This could be done using define and that'd reduce the complexity of the config quite a bit on the server side.

Reply:
I don't think i've ever heard of define...do you have a link to some resource i can look into?